Error when generating a library

```
npm ERR! peerinvalid The package karma does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer karma-jasmine@0.0.3 wants karma@~0.9
npm ERR! peerinvalid Peer karma-chrome-launcher@0.0.2 wants karma@~0.9.3
npm ERR! peerinvalid Peer karma-firefox-launcher@0.1.3 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-html2js-preprocessor@0.1.0 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-requirejs@0.2.2 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-script-launcher@0.1.0 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-coffee-preprocessor@0.2.1 wants karma@>=0.11.14
npm ERR! peerinvalid Peer karma-phantomjs-launcher@0.1.4 wants karma@>=0.9

npm ERR! System Darwin 13.1.0
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! ...
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.10
npm ERR! code EPEERINVALID
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR! ...
npm ERR! not ok code 0
```

--- a/app/templates/_bower.json
+++ b/app/templates/_bower.json
@@ -15,13 +15,13 @@
         "**/.*"
     ],
     "dependencies": {
-        "angular": "~1.0.8"<% if (config.includeAngularModuleResource) { %>,
-        "angular-resource": "~1.0.8"<% } %><% if (config.includeAngularModuleCookies) { %>,
-        "angular-cookies": "~1.0.8"<% } %><% if (config.includeAngularModuleSanitize) { %>,
-        "angular-sanitize": "~1.0.8"<% } %>
+        "angular": "~1.2.21"<% if (config.includeAngularModuleResource) { %>,
+        "angular-resource": "~1.2.21"<% } %><% if (config.includeAngularModuleCookies) { %>,
+        "angular-cookies": "~1.2.21"<% } %><% if (config.includeAngularModuleSanitize) { %>,
+        "angular-sanitize": "~1.2.21"<% } %>
     },
     "devDependencies": {
-        "angular-mocks": "~1.0.8",
-        "angular-scenario": "~1.0.8"
+        "angular-mocks": "~1.2.21",
+        "angular-scenario": "~1.2.21"
     }
 }

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -7,17 +7,17 @@
     },
     "dependencies": {},
     "devDependencies": {
-        "grunt": "~0.4.1",
-        "grunt-contrib-concat": "~0.3.0",
-        "grunt-contrib-uglify": "~0.2.0",
-        "grunt-contrib-jshint": "~0.6.0",
-        "grunt-contrib-connect": "~0.3.0",
-        "grunt-contrib-watch": "~0.4.0",
-        "grunt-karma": "~0.4.3",
-        "connect-livereload": "~0.2.0",
-        "karma": "~0.9",
-        "karma-jasmine": "~0.0.3",
-        "karma-chrome-launcher": "~0.0.2"
+        "grunt": "~0.4.5",
+        "grunt-contrib-concat": "~0.5.0",
+        "grunt-contrib-uglify": "~0.5.1",
+        "grunt-contrib-jshint": "~0.10.0",
+        "grunt-contrib-connect": "~0.8.0",
+        "grunt-contrib-watch": "~0.6.1",
+        "grunt-karma": "~0.8.3",
+        "connect-livereload": "~0.4.0",
+        "karma": "~0.12",
+        "karma-jasmine": "~0.1.5",
+        "karma-chrome-launcher": "~0.1.4"
     },
     "engines": {
         "node": ">=0.8.0"

